tests_e2e/orchestrator/runbook.yml

```diff
@@ -54,6 +54,8 @@ variable:
       - publish_hostname
       - recover_network_interface
       - cgroup_v2_disabled
+      - agent_wait_for_cloud_init
```

Comment:
there is no point in adding those to the daily run, is there? the custom image has not changed

Reply:
Last time when I updated lisa, I didn't know we have this test to validate against new lisa. I sent pr without checking it.  Few later days I figured we have this test, and test was not passing on new lisa since lisa changed schema in templates. So custom template we use in this test is failing. That's why I added this as part of normal run to catch it

Reply:
good catch, though

Reply:
Whole point I want to make here if we treat them special people tend to forget since they are not part of regular run. To run them we don't need any special settings, one could run like regular test_suite, pass test_suit list in pipeline or to the runbook now. 

But I agree adding to daily run is overhead when custom image has not changed. It's appropriate to add when we build custom-images on every run. I'll add a note and comment it out here.